mandatory/4-tax.js

```diff
@@ -17,8 +17,12 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
 
+
+
+function addTaxAndFormatCurrency(x) {
+  return (x + 0.20*x).toFixed(2);
```

Comment:
Hi Deago, well done for fixing it. If you want to add a pound sign, think of it as a "string" that you add with + in your return formula.

Reply:
It can be rewritten like this:
```
return (1.2 * x).toFixed(2);
```